app/Conference.php

```diff
@@ -109,6 +121,16 @@ public function scopeOpenCfp($query)
             ->where('cfp_ends_at', '>', Carbon::now());
     }
 
+    public function scopeApproved($query)
```

Comment:
I wonder if this should be a global scope. I'll keep reading and see... but I bet we seldom want to see un-approved conferences.

Reply:
you mean we should use a global scope for approved conferences cause we mostly never use un-approved ones? Hm, difficult to tell for me. Personally, I prefer not to use global scopes but maybe it makes sense here. Your decision :-)